ci/run_envoy_docker_windows.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+# TODO(sunjayBhatia, wrowe): update this note once we have an RBE toolchain generated for Windows
+# NOTE: Update this from the latest pushed image here: https://hub.docker.com/r/envoyproxy/envoy-build-windows2019/tags
+ENVOY_BUILD_SHA="3cbc11e373dc4e3a523b9273ed010c5e0f197874"
```

Comment:
This line is mirroring https://github.com/envoyproxy/envoy/blob/f0ebefc2a36746dbf12301d3c258e2f1b66cfe82/.bazelrc#L162 but we don't have a toolchain generated at the moment in envoy-build-tools so this is the tag of the latest built image pushed to docker hub